ArcadiaParties-SPA/src/app/app.module.ts

```diff
@@ -2,26 +2,34 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 import { StoreModule } from '@ngrx/store';
 import { StoreDevtoolsModule } from '@ngrx/store-devtools';
+import { MaterialModule } from './material/material.module';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
 import { AppComponent } from './app.component';
 import { ROOT_REDUCERS } from './reducers/reducers';
 import { environment } from '../environments/environment';
+import { HeaderComponent } from './header/header.component';
+import { ContentComponent } from './content/content.component';
 
 @NgModule({
-  declarations: [
-    AppComponent
-  ],
-  imports: [
-    BrowserModule,
-    StoreModule.forRoot(ROOT_REDUCERS, {
-      runtimeChecks: {
-        strictStateImmutability: true,
-        strictActionImmutability: true
-      }
-    }),
-    StoreDevtoolsModule.instrument({ maxAge: 25, logOnly: environment.production })
-  ],
-  providers: [],
-  bootstrap: [AppComponent]
+   declarations: [
```

Comment:
This file had indent by 2 spaces and now it has 3 spaces.

Reply:
These changes was at first commit in this PR. That's mean only differences between Duveshka's workspace and mine.

Reply:
Yes, they were added in your commit. Correct indent is 2 spaces (or 4 spaces), but you somehow made it 3.

Reply:
Yes, they were added in your commit. Correct indent is 2 spaces (or 4 spaces), but you somehow made it 3.

Reply:
Ok, I'll use Prettier. Problem has gone.

Reply:
Then add editor config file to specify project settings for Prettier. Something like this:
```
# Editor configuration, see https://editorconfig.org
root = true

[*]
charset = utf-8
max_line_length = 120
end_of_line = crlf
trim_trailing_whitespace = true
insert_final_newline = true
indent_style = space
indent_size = 2
quote_type = single
```

Reply:
And also add Prettier extension for VS Code and set default code formatter: https://github.com/prettier/prettier-vscode#using-command-palette-cmdctrl--shift--p

Reply:
indent configured.